src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1414,14 +1426,14 @@ end
 
 variables {G}
 
-lemma walk.length_eq_of_mem_finset_walk_length {n : ℕ} {u v : V} (p : G.walk u v) :
-  p ∈ G.finset_walk_length n u v → p.length = n :=
-(set.ext_iff.mp (G.coe_finset_walk_length_eq n u v) p).mp
+lemma walk.mem_finset_walk_length_iff_length_eq {n : ℕ} {u v : V} (p : G.walk u v) :
+  p ∈ G.finset_walk_length n u v ↔ p.length = n :=
+set.ext_iff.mp (G.coe_finset_walk_length_eq n u v) p
 
 variables (G)
 
 instance fintype_set_walk_length (u v : V) (n : ℕ) : fintype {p : G.walk u v | p.length = n} :=
-fintype.subtype (G.finset_walk_length n u v) $ λ p,
+fintype.of_finset (G.finset_walk_length n u v) $ λ p,
```

Comment:
What does this change concretely?

Reply:
`fintype.subtype` and `fintype.of_finset` are definitionally equal, but syntactically the first is for `subtype` and the second is for a coerced set. Similarly for `fintype.card_of_subtype` and `fintype.card_of_finset` below.